diff --git a/src/generators/format-files.js b/src/generators/format-files.js
index 9fc3fe5da89c97f1434856c19b71ffa21c18ee07..9d99357ebf77109e325670ed728296a1ea32cf32 100644
--- a/src/generators/format-files.js
+++ b/src/generators/format-files.js
@@ -15,7 +15,7 @@ function formatFiles(tree) {
     return tslib_1.__awaiter(this, void 0, void 0, function* () {
         let prettier;
         try {
-            prettier = yield Promise.resolve().then(() => require('prettier'));
+            prettier = yield Promise.resolve().then(() => require('prettier-2'));
         }
         catch (_a) { }
         sortTsConfig(tree);